salt/states/pkgrepo.py

```diff
@@ -446,8 +446,18 @@ def managed(name, ppa=None, copr=None, aptkey=True, **kwargs):
     # out of the state itself and into a module that it makes more sense
     # to use. Most package providers will simply return the data provided
     # it doesn't require any "specialized" data massaging.
-    if "pkg.expand_repo_def" in __salt__:
-        sanitizedkwargs = __salt__["pkg.expand_repo_def"](repo=repo, **kwargs)
+    if __grains__.get("os_family") == "Debian":
```

Comment:
My concern with this change, is what if the `__virtual__` function in salt/modules/aptpkg.py is changed to adjust what OS's can load the aptpkg module and hence the expand_repo_def function. We would need to ensure we update the virtual function and this if logic here. I think that is the advantage with having the function loaded into `__salt__`.

Reply:
But the function has no use on the CLI...
It should actually just be an util, I just didn't want to create yet another utils module just for a single function.